editor/editor_data.cpp

```diff
@@ -556,7 +556,7 @@ bool EditorData::check_and_update_scene(int p_idx) {
 
 	bool must_reload = _find_updated_instances(edited_scene[p_idx].root, edited_scene[p_idx].root, checked_scenes);
 
-	print_line("MUST RELOAD? " + itos(must_reload));
+	print_line(TTR("MUST RELOAD? ") + itos(must_reload));
```

Comment:
Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.